tests/test_client.py

```diff
@@ -44,3 +48,71 @@ def test_save_log_when_published_called(self, caplog):
                 'topic': 'order-cancelled',
             }
         }
+
+
+class TestSubscriber:
+
+    @pytest.fixture()
+    def project_id(self):
+        return settings.RELE_GC_PROJECT_ID
+
+    @pytest.fixture()
+    def subscriber(self, project_id):
+        credentials = settings.RELE_GC_CREDENTIALS
+
+        return Subscriber(project_id, credentials)
+
+    @patch.object(SubscriberClient, 'create_subscription')
+    def test_create_subscription_without_ack_deadlines(
```

Comment:
```suggestion
    def test_creates_subscription_with_60s_ack_deadline_when_none_provided(
```

Reply:
Thanks for the suggestion, this better describes the test case. I only have one doubt: do you see useful to hardcode the expected value in the name of the test? That could be misleading in case this value changes.

Reply:
I agree hard coding would not be wanted in this test. Instead, we are kind of documenting the default value in the test when no `ack_deadline` is set.